src/main/java/nl/dtls/fairdatapoint/config/AclConfig.java

```diff
@@ -58,19 +59,15 @@
     private AclRepository aclRepository;
 
     @Bean
-    public CacheManager cacheManager() {
-        return new ConcurrentMapCacheManager(CACHE_NAME);
-    }
-
-    @Bean
-    public AclCache aclCache() {
-        Cache springCache = cacheManager().getCache(CACHE_NAME);
+    public AclCache aclCache(ConcurrentMapCacheManager cacheManager) {
```

Comment:
Any reason to go for the specific `ConcurrentMap` implementation over the parent `CacheManager` interface for injection?

Reply:
Yes, because you can't create new caches with `CacheManager`

Reply:
The javadoc of the `CacheManager.getCache()` interface states:
> Note that the cache may be lazily created at runtime if the native provider supports it.

I don't see any explicit invocations of the `ConcurrentMapCacheManager` method `createConcurrentMapCache(String name)` in this PR 😕

Reply:
Because it's hidden in a `setCacheNames ` method 😃